lib/streamlit/elements/arrow.py

```diff
@@ -73,6 +75,15 @@ def _arrow_dataframe(
             This takes precedence over the width argument.
             This argument can only be supplied by keyword.
 
+        num_rows : int or None
```

Comment:
I think we don't need to have this as a parameter here since it is anyways just used internally. We usually are very cautious about adding new arguments to our commands.  But maybe this is planned to be supported via the API (@jrieke)?